### ISSUE TITLE
Wrong "up" detection

### ISSUE THREAD

**Issue by kojid0** [WBerredo/IsItUP#6]
I get several "Website is up" messages even if the website is still down. Then, the bot sends you a "Website is down" message after 4-5 minutes.

Maybe the bot should check twice (like one now and the second time after 30sec/1min) before sending wrong notifications to the users...

**Comment by JosephineSharp** [comment]
I get "Website is down" messages even if the website is up. 🤔

**Comment by WBerredo** [comment]
Yes, I am getting this sometime, but I am without free time right now.
I will fix this when I get a good time.

**Comment by harked** [comment]
@WBerredo does this issue fixed already?

**Comment by WBerredo** [comment]
Hi @harked, unfortunately I am dedicating my time for some other projects and this project is not on my plans for the next months, sorry about that. That said, I am always open to receive and review pull requests for all projects.
